resin/models/application.py

```diff
@@ -488,3 +488,34 @@ def generate_provisioning_key(self, app_id):
             'POST',
             endpoint=self.settings.get('api_endpoint')
         )
+
+    def set_to_release(self, app_id, commit_id):
+        """
+        Set an application to a specific commit.
+
+        Args:
+            app_id (str): application id.
+            commit_id (str) : commit id.
+
+        Returns:
+            OK/error.
+
+        Examples:
+            >> > resin.models.application.set_to_release('5685', '7dba4e0c461215374edad74a5b78f470b894b5b7')
+            'OK'
+
+        """
+
+        params = {
+            'filter': 'id',
+            'eq': app_id
+        }
+
+        data = {
+            'commit': commit_id
```

Comment:
There is also the `should_track_latest_release` boolean that should be set to `false`, otherwise the `commit` will get updated on the next push.
We also need a way to set this back to `true`.
See: https://github.com/resin-io-playground/staged-releases

Reply:
Oh I just saw that there is a dedicated method for that.

Reply:
👍